libsolidity/formal/BMC.cpp

```diff
@@ -61,15 +62,14 @@ BMC::BMC(
 
 void BMC::analyze(SourceUnit const& _source, map<ASTNode const*, set<VerificationTargetType>, smt::EncodingContext::IdCompare> _solvedTargets)
 {
-	if (m_interface->solvers() == 0)
+	/// At this point every enabled solver is available.
```

Comment:
```suggestion
	// At this point every enabled solver is available.
```

Actually, I just noticed that you're using these docstring comments in many places in implementation files. Why?

Reply:
Yea, no idea lol